src/component/common/InstanceStatus/InstanceStatus.tsx

```diff
@@ -0,0 +1,22 @@
+import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
+import React, { ReactNode } from 'react';
+import { InstanceStatusBar } from 'component/common/InstanceStatus/InstanceStatusBar';
+
+interface IInstanceStatusProps {
+    children: ReactNode;
+}
+
+export const InstanceStatus = ({ children }: IInstanceStatusProps) => {
```

Comment:
Wouldn't it be clearer to have this as `export const InstanceStatus: FC = ({ children }) => {` and remove the props interface?